docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
+
+* **SCHEME**: The transport scheme, e.g. **file://** or **https://**
+* **URI**: For remote transport schemes, the signature server URI, e.g. **example.com**
+* **PATH_PREFIX**: The path to the base of the registry directory
+* **REGISTRY**: The registry URI and optional port, e.g. **registry.example.com:5000**
+* **REPOSITORY**: The repository namespace. May occur multiple times for registries that support multiple repository namespaces.
+* **IMAGE**: The name of the image
+* **MANIFEST_DIGEST**: The value of the manifest digest, including the hash function and hash, e.g. **sha256:HASH**
+* **INT**: An integer of the signature starting with 1. For multiple signatures increment by 1, e.g. **signature-1**, **signature-2**.
```

Comment:
Just a note: this differs from https://github.com/aweiteka/image/blob/sigspec/signature/spec/README.md#static-file-layout